components/shared/Sharing.vue

```diff
@@ -119,7 +120,11 @@ export default class Sharing extends Vue {
 
   private active = false
 
-  private hashtags = 'KusamaNetwork,KodaDot'
+  public hashtags = 'KusamaNetwork,KodaDot'
+
+  get historyStore() {
+    return useHistoryStore()
+  }
```

Comment:
```suggestion
  private historyStore = useHistoryStore()
```
I guess there's no need to repeatedly call the getter function every time we access the store.

Reply:
hmmm... does it work? didn't tried like that

Reply:
Yes, it would work. Let's give it a shot